README.md

```diff
@@ -41,23 +41,42 @@ Then you need to switch to Java8 and run the command to generate binaries for Kt
 
 Then build and run the Integration Tests using the `its` property:
 
-    ./gradlew build -Pits --info --no-daemon -Dsonar.runtimeVersion=7.9
+    ./gradlew build -Pits --info --no-daemon -Dsonar.runtimeVersion=9.7.1.62043
 
 You can also build and run only Ruling Tests using the `ruling` property:
 
-    ./gradlew build -Pruling --info --no-daemon -Dsonar.runtimeVersion=7.9
+    ./gradlew build -Pruling --info --no-daemon
 
 You can also build and run only Plugin Tests using the `plugin` property:
 
-    ./gradlew build -Pplugin --info --no-daemon -Dsonar.runtimeVersion=7.9
+    ./gradlew build -Pplugin --info --no-daemon -Dsonar.runtimeVersion=9.7.1.62043
 
 To run e.g. the ruling tests in the IDE, create a new Run/Debug Configuration where you run the following:
 
-    :its:ruling:test -Pruling -Dsonar.runtimeVersion=7.9
+    ./gradlew :its:ruling:test --info -Pruling
 
 You can also run single ruling tests, e.g.:
 
-    :its:ruling:test --tests "org.sonarsource.slang.SlangRulingTest.test_kotlin_corda" -Pruling -Dsonar.runtimeVersion=7.9
+    ./gradlew :its:ruling:test --info -Pruling --tests "org.sonarsource.slang.SlangRulingTest.test_kotlin_corda"
```

Comment:
Same here, we are talking about how to configure the IDE, not how to use gradle directly